<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuAcceso</class>
 <widget class="QDialog" name="MenuAcceso">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>606</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>7</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Lanzador de paquetes del menu del escritorio</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recoverdrake.qrc">
    <normaloff>:/Imagenes/tirachinas.png</normaloff>:/Imagenes/tirachinas.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Navegacion</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="1" column="0" colspan="4">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>Filtrar</string>
        </property>
        <property name="icon">
         <iconset resource="recoverdrake.qrc">
          <normaloff>:/Imagenes/embudo.png</normaloff>:/Imagenes/embudo.png</iconset>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Eliminar</string>
        </property>
        <property name="icon">
         <iconset resource="recoverdrake.qrc">
          <normaloff>:/Imagenes/bad.png</normaloff>:/Imagenes/bad.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton_7">
        <property name="text">
         <string>Nuevo</string>
        </property>
        <property name="icon">
         <iconset resource="recoverdrake.qrc">
          <normaloff>:/Imagenes/borrar.png</normaloff>:/Imagenes/borrar.png</iconset>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Salir</string>
        </property>
        <property name="icon">
         <iconset resource="recoverdrake.qrc">
          <normaloff>:/Imagenes/puerta.png</normaloff>:/Imagenes/puerta.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Modificar</string>
        </property>
        <property name="icon">
         <iconset resource="recoverdrake.qrc">
          <normaloff>:/Imagenes/sustituir.png</normaloff>:/Imagenes/sustituir.png</iconset>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="4">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Insertar</string>
        </property>
        <property name="icon">
         <iconset resource="recoverdrake.qrc">
          <normaloff>:/Imagenes/good.png</normaloff>:/Imagenes/good.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>501</width>
       <height>211</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Caracteristicas del Lanzador</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="6" column="0" colspan="3">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Comentario en espanol</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit_6"/>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QTextEdit" name="textEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Categoria</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_6">
        <property name="maximumSize">
         <size>
          <width>92</width>
          <height>26</height>
         </size>
        </property>
        <property name="text">
         <string>Lanzar</string>
        </property>
        <property name="icon">
         <iconset resource="recoverdrake.qrc">
          <normaloff>:/Imagenes/thumbs_up.png</normaloff>:/Imagenes/thumbs_up.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nombre</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Ejecutable</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Codigo</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Nombre en espanol</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="3">
       <widget class="QTextEdit" name="textEdit_4">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Comentario</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QTextEdit" name="textEdit_3">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QTableView" name="tableView">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>702</x>
     <y>494</y>
     <width>124</width>
     <height>71</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>124</width>
     <height>71</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>124</width>
     <height>71</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="recoverdrake.qrc"/>
 </resources>
 <connections/>
</ui>
